packages/terra-date-input/tests/wdio/date-input-field-spec.js

```diff
@@ -162,283 +141,281 @@ Terra.describeViewports('Date Input', ['medium'], () => {
         browser.click('select[name="terra-date-month-date-input"]');
         browser.keys(['Enter']);
         browser.keys(['Delete']);
-      });
 
-      Terra.it.matchesScreenshot();
+        Terra.validates.element('default');
+      });
     });
   });
 
   describe('Date Input Field Day input keyboard operations', () => {
     describe('Day Input UP_ARROW increments day by 1 when day value is filled in', () => {
-      before(() => {
-        browser.url('/#/raw/tests/terra-date-input/date-input/populated-date-input-field');
+      it('should increment day by 1', () => {
+        browser.url('/raw/tests/terra-date-input/date-input/populated-date-input-field');
         browser.refresh();
         // Removes the blinking cursor to prevent screenshot mismatches.
         Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
         Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
 
         browser.click('input[name="terra-date-day-date-input"]');
         browser.keys(['ArrowUp']);
-      });
 
-      Terra.it.matchesScreenshot();
+        Terra.validates.element('default');
+      });
     });
 
     describe('Day Input UP_ARROW increments day by 1 when day value is empty', () => {
-      before(() => {
-        browser.url('/#/raw/tests/terra-date-input/date-input/default-date-input-field');
+      it('should increment day by 1', () => {
+        browser.url('/raw/tests/terra-date-input/date-input/default-date-input-field');
         browser.refresh();
         // Removes the blinking cursor to prevent screenshot mismatches.
         Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
         Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
 
         browser.click('input[name="terra-date-day-date-input"]');
         browser.keys(['ArrowUp']);
+        Terra.validates.element('default');
       });
-
-      Terra.it.matchesScreenshot();
     });
 
     describe('Day Input UP_ARROW is cycles to 01 when the day has reached 31', () => {
-      before(() => {
-        browser.url('/#/raw/tests/terra-date-input/date-input/default-date-input-field');
+      it('should cycle to 1', () => {
+        browser.url('/raw/tests/terra-date-input/date-input/default-date-input-field');
         browser.refresh();
         // Removes the blinking cursor to prevent screenshot mismatches.
         Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
         Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
 
         browser.click('input[name="terra-date-day-date-input"]');
         browser.keys(['31', 'ArrowUp']);
-      });
 
-      Terra.it.matchesScreenshot();
+        Terra.validates.element('default');
```

Comment:
The screenshot name currently is optional and will use "default" as the name if not provided. WDIO6 will make the screenshot name a required argument. We should give these a more meaning screenshot name instead of "default".

Reply:
Done fe2608b